packages/calendar/test/lion-calendar.test.js

```diff
@@ -1393,6 +1393,8 @@ describe('<lion-calendar>', () => {
       ]);
 
       localize.locale = 'nl-NL';
+      // await localize.loadingComplete;
```

Comment:
why would  `localize.loadingComplete` not work here?

Reply:
Ah right, that was a remnant of me trying to check for all failing tests whether they are timing related. Updated it.